jsonnet/prometheus.jsonnet

```diff
@@ -54,6 +54,27 @@ local namespacesRole = policyRule.new() +
       },
     },
 
+    routeConsole:{
```

Comment:
the openshift console actually proxies the requests into the cluster, so we don't need this route